app/include/user_config.h

```diff
@@ -52,7 +52,9 @@ extern void luaL_assertfail(const char *file, int line, const char *message);
 
 #define ICACHE_STORE_TYPEDEF_ATTR __attribute__((aligned(4),packed))
 #define ICACHE_STORE_ATTR __attribute__((aligned(4)))
-#define ICACHE_RAM_ATTR __attribute__((section(".iram0.text")))
+#define STRINGIZE(x) STRINGIZE2(x)
```

Comment:
Defines in `user_confgi.h` in practice end up getting included in most modules and platform files, so should these be prefixed to remove the risk of a collision with another use of `STRINGIZE`, e.g.
-  `STRINGIZE`->`ICACHE_STRINGIZE`
-  `STRINGIZE2`->`ICACHE_STRINGIZE2`